cadc-download-manager-server/src/main/resources/META-INF/resources/chooser.jsp

```diff
@@ -72,11 +72,13 @@
         "http://www.w3.org/TR/html4/loose.dtd">
 
 <%@ taglib uri="WEB-INF/c.tld" prefix="c"%>
-
+<%@ page import="ca.nrc.cadc.config.ApplicationConfiguration" %>
 <%@ page import="ca.nrc.cadc.dlm.server.ServerUtil" %>
 <%@ page import="ca.nrc.cadc.dlm.server.DispatcherServlet" %>
 <%@ page import="ca.nrc.cadc.dlm.server.UrlListServlet" %>
 <%
+    final ApplicationConfiguration configuration = new ApplicationConfiguration();
```

Comment:
This is the constructor that does not take a configuration file as an argument.  Is it being read in a different way?